addons/addon-base-raas/packages/base-raas-services/lib/study/study-service.js

```diff
@@ -388,7 +388,8 @@ class StudyService extends Service {
       throw this.boom.forbidden('Permissions cannot be set for studies in the "Open Data" category', true);
     }
 
-    if (isMyStudies(studyEntity)) {
+    // If study entity is My Study and not part of a migration request, throw error
+    if (isMyStudies(studyEntity) && updateRequest.usersToRemove[0].uid !== '*') {
```

Comment:
when updating permission we can now use `*`  for any update permissions call; is that the intention?

Also we are saying MyStudies MUST have  `*` in "toRemove" is that the intention?

Reply:
By design, we have never before allowed changing permissions on My Study because they are not sharable. So, yes, MyStudies MUST have `*` in "toRemove" because that is what the API controller passes in. It is the only API that does this.
Theoretically, a user could put `*` in `userToRemove` in an update permissions API call body which might cause unexpected behavior. We can either assume that won't happen because `*` has never been a valid input before or we can build some sort of check to make sure that the only API that can pass `*` to update permissions in `userToRemove` is the my studies migration API. What is acceptable/required in your opinion?

Reply:
I would put in logic to prevent the use of `*` unless we explicitly want that functionality

Reply:
Got it! Will do.